src/osd/OSD.h

```diff
@@ -1868,9 +1868,11 @@ class OSD : public Dispatcher,
   // -- failures --
   std::map<int,utime_t> failure_queue;
   std::map<int,std::pair<utime_t,entity_addrvec_t> > failure_pending;
+  std::map<int,entity_addrvec_t> failure_immediate_pending;
```

Comment:
Why do we need this as well as failure_pending?  Seems like they are essentially the same?